.gitignore

```diff
@@ -3,3 +3,5 @@
 build/
 *.iml
 /local.properties
+/plugin/out
```

Comment:
Nit: To emphasize that we intend to match directories, we usually use a trailing slash and sort them in after the other directories at the beginning of this file, i.e. after `/.idea/`.

Reply:
Done.